app/scripts/views/settings/avatar_change.js

```diff
@@ -61,7 +54,11 @@ function ($, _, Cocktail, FormView, AvatarMixin, SettingsMixin, Template, AuthEr
 
     remove: function () {
       var self = this;
-      return self.getSignedInAccount().deleteAvatar(self.avatarId)
+      var account = self.getSignedInAccount();
+      return account.getAvatar()
```

Comment:
Does `getAvatar` mean `get the current avatar`? If so, should the name be updated to reflect its purpose? Can `account.deleteAvatar` be called without the initial call to `account.getAvatar`? I'm asking myself whether the API can be cleaned up a bit.